platform/platform-app/src/main/resources/features.xml

```diff
@@ -835,6 +835,7 @@
     <feature name="mime-core" install="manual" version="${project.version}"
              description="DDF MIME Core">
         <feature>mime-core-api</feature>
+        <feature>catalog-core-api</feature>
```

Comment:
No you can't do that, that'll create a circular dependency. I was meaning that we should just use the same tika-bundle that catalog is trying to use. Karaf will only pull it in once so it will essentially ignore the line in catalog, if the mime-core feature was already installed.